src/App.tsx

```diff
@@ -1,10 +1,43 @@
 /* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import { useEffect, useState } from 'react';
 import { UserWarning } from './UserWarning';
-import { USER_ID } from './api/todos';
+import { USER_ID, getTodos } from './api/todos';
+import { Footer } from './components/Footer/Footer';
+import { Header } from './components/Header/Header';
+import { TodoList } from './components/TodoList/TodoList';
+import { Todo } from './types/Todo';
+import { Errors } from './components/Errors/Errors';
+import { TodoStatus } from './types/TodoStatus';
+
+const getVisibleTodos = (todos: Todo[], status: TodoStatus) => {
+  let visibleTodos = [...todos];
+
+  if (status !== 'All') {
```

Comment:
also here can use TodoStatus

```suggestion
  if (status !== TodoStatus.all) {
```